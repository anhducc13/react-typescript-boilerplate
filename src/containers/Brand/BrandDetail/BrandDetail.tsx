import React from 'react';
import { t } from 'helpers/i18n';
import AppContainer from 'containers/AppLayout/AppContainer';
import ContentBlock from 'components/shared/ContentBlock';

const BrandDetail: React.FC = () => {
  return (
    <AppContainer title={t('BrandDetail')}>
      <ContentBlock>{t('BrandDetail')}</ContentBlock>
    </AppContainer>
  );
};

export default BrandDetail;
