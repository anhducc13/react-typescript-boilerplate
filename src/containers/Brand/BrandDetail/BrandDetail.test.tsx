import React from 'react';
import { render } from '@testing-library/react';
import BrandDetail from './BrandDetail';

it('renders correctly', () => {
  const { container } = render(<BrandDetail />);
  expect(container.firstChild).toMatchSnapshot();
});
