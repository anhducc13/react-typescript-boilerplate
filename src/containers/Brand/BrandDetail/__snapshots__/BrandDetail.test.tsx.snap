// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  class="app-container"
>
  <div
    class="app-container-head"
  >
    <h4
      class="ant-typography app-title"
    >
      Thông tin thương hiệu
    </h4>
  </div>
  <div
    class="app-container-body"
  >
    <div
      class="ant-card content-block"
    >
      <div
        class="ant-card-body"
      >
        Thông tin thương hiệu
      </div>
    </div>
  </div>
</div>
`;
